packages/shared-ux/markdown/mocks/jest.config.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
```

Comment:
Do we need jest config files in the "mocks" package?

cc @clintandrewhall

Reply:
I don't think so! I deleted it and the jest* reference in the tsconfig, thanks!